src/DatePicker/DatePicker.test.js

```diff
@@ -175,20 +232,25 @@ describe('<DatePicker />', () => {
 
         wrapper.find('input[type="text"]').simulate('keypress', { key: 'Enter' });
 
+
+        // press Esc key
+        wrapper.find('input[type="text"]').simulate('keypress', { key: 'Esc' });
```

Comment:
Shouldn't each `test` block end with an assertion?

Reply:
Added assertions at the end of test